nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -152,7 +152,8 @@ language governing permissions and limitations under the License. -->
         </dependency>
         <dependency>
             <groupId>org.apache.activemq</groupId>
-            <artifactId>activemq-client</artifactId>
+            <artifactId>activemq-all</artifactId>
+            <scope>test</scope>
```

Comment:
I agree we don't need the activemq-all at runtime, but I think we do need the activemq-client at runtime, no? As-is here, we bring it all in for test and none of it at runtime.